README.md

```diff
@@ -0,0 +1,99 @@
+[![GoDoc](https://godoc.org/github.com/blocky/parlor?status.svg)](https://godoc.org/github.com/blocky/parlor)
+[![Build Status](https://app.travis-ci.com/blocky/parlor.svg?token=JczzdP6eMqmEqysZ8pDf&branch=main)](https://app.travis-ci.com/blocky/parlor)
+[![Go Report Card](https://goreportcard.com/badge/github.com/blocky/parlor)](https://goreportcard.com/report/github.com/blocky/parlor)
+
+# Parlor - opinionated testing tools
+
+Parlor is Blocky's opinionated set of testing tools that augment
```

Comment:
I originally chose the name "parlor" because of the name "suite"

A suite is a set of rooms, and Testify's Suite has a lot of tools, hence a "suite"

Parlor is an antiquated term for a a reception & conversation room. And since we are adding to one of the "rooms" of the "suite", the suite tool, i thought the extra "furnishing" we were adding would result in a "parlor", since parlors used to be very well furnished because of their original purpose

if the name parlor causes any confusion maybe this clears it up

Reply:
Thank you!!